app/frontend/src/registry/components/search/SearchTable.vue

```diff
@@ -181,22 +171,44 @@ export default {
       'activity'
     ])
   },
+  watch: {
+    // When drillers has a new state, scroll to the top of the searchTable.
+    drillers () {
+      this.scrollToTableTop()
+    }
+  },
   methods: {
+    /**
+     * Gets called when the user clicks on the next button, load the next result page.
+     */
     paginationNext () {
       // API provides 'next' and 'previous' links with query strings for the current search
-      if (this.drillers.next && ~this.drillers.next.indexOf('?')) {
-        const q = this.drillers.next.split('?')[1]
-        this.$store.dispatch(FETCH_DRILLER_LIST, querystring.parse(q))
-      }
+      if (this.drillers.next && ~this.drillers.next.indexOf('?'))
+        this.getPage(this.drillers.next.split('?')[1])
     },
+    /**
+     * Gets called when the user clicks on the previous button, load the previous result page.
+     */
     paginationPrev () {
-      if (this.drillers.previous && ~this.drillers.previous.indexOf('?')) {
-        const q = this.drillers.previous.split('?')[1]
-        this.$store.dispatch(FETCH_DRILLER_LIST, querystring.parse(q))
-      }
+      if (this.drillers.previous && ~this.drillers.previous.indexOf('?'))
+        this.getPage(this.drillers.previous.split('?')[1])
+    },
+    /**
+     * Triggers FETCH_DRILLER_LIST store action.
+     * @param {string} query QueryString of the new page to be loaded.
+     */
+    getPage (query) {
+      if(!query) throw new Error('query parameter is required.')
```

Comment:
Add space after `if`
e.g.`if(!query)` to `if (!query)` (just another linter warning)

Reply:
thanks @stephenhillier for the code review, I really enjoy reading your comments and that things have to be precise ;-). I can do the changes tonight and commit on the same branch if you want.